.github/workflows/report-stable.yml

```diff
@@ -0,0 +1,24 @@
+
+name: Report-API-Updates
+
+# Run tests on pull requests and when changes are directly
+# commited to master.
+on:
+  workflow_dispatch: {}
+
+jobs:
+  find-updates:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Run makefile
+      run: make api-report-updates  RESULTS_DIR=_results
```

Comment:
Don't we need _implements.json_ prior to `api-report-updates` ?

Reply:
No, the function only needs the doc/api-status.json file to do it work. On top of that, the make file rules that *do* require implmenets.json to exist have to internally depend on the "implments-json" target, so it wouldn't show in the yaml workflow anyway.

Reply:
I asked because of the following error I noticed while running `make api-report-updates` and now realized that we have general check for _./_results/implements.json_ in _main()_:
```
$ CEPH_VERSION=quincy RESULTS_DIR=_results make api-report-updates 
mkdir -p _results
./contrib/apiage.py --mode=find-updates \
	--current-tag="$(git describe --tags --abbrev=0)" \
	> _results/updates-found.json
error: no source data found (path: ./_results/implements.json)
make: *** [Makefile:280: api-report-updates] Error 1
```

Relevant code block:
```
    cli = parser.parse_args()

    api_src = read_json(cli.source) if cli.source else {}
    api_tracked = read_json(cli.current) if cli.current else {}

    if not api_src:
        print(
            f"error: no source data found (path: {cli.source})", file=sys.stderr
        )
        sys.exit(1)
```

Reply:
Oh, it shouldn't be doing that. Thanks for pointing it out.

Reply:
Updated.